doc/source/conf.py

```diff
@@ -31,7 +31,7 @@
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
-extensions = ["sphinx.ext.autodoc", "autoapi.extension"]
+extensions = ['sphinxcontrib.fulltoc', "sphinx.ext.autodoc", "autoapi.extension", 'sphinx_toolbox.code',]
```

Comment:
The documentation build seems to crash because sphinxcontrib is not installed https://readthedocs.org/projects/turbo-django/builds/14485083/

Reply:
https://github.com/hotwire-django/turbo-django/pull/39 contains the fix.